coordination.go

```diff
@@ -38,9 +41,9 @@ func touchFile(path string) error {
 // LockCoordinationDir takes an exclusive lock on the given coordination
 // directory. It returns a coordinator that holds the lock and may be used to
 // manipulate the directory. If the directory is already locked, the function will block until the lock can be acquired.
```

Comment:
This line is long. It would be fine if it wasn't clear you already broke the previous lines for being too long.

Reply:
I just realized you almost certainly didn't write this, so disregard.

Reply:
I did, the whole `coordination.go` file is on me. Just forgot to hit my vim comment-wrap button I think.